01-main/packages/tixati

```diff
@@ -1,7 +1,7 @@
 DEFVER=1
-get_website "https://www.tixati.com/download/linux.html"
+get_website "https://www.tixati.com/linux"
 if [ "${ACTION}" != "prettylist" ]; then
-    URL="$(grep "amd64\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f2)"
+    URL=$(grep -m 1 -o  '\http.*amd64\.deb.asc'  /var/cache/deb-get/tixati.html  | sed 's/.asc//')
```

Comment:
It pedantic, but I think this wants to be
`URL=$(grep -m 1 -o 'http.*amd64\.deb\.asc'  /var/cache/deb-get/tixati.html  | sed 's/\.asc//')`
to escape the two periods.
The backslash before the h is a typo, yes?

Reply:
yes (it was the one for .asc but this laptop old likes to move cursors around for fun and I wasn't looking where I was typing clearly :-) 
Pedantic is my middle name :-)